app/src/main/res/values/strings.xml

```diff
@@ -50,16 +50,11 @@
     <!-- Content of about focus. Argument 1 is the app name (Firefox Focus/Firefox Klar/etc).
          Argument 2 is the url for the learn-more link. -->
     <string name="about_content"><![CDATA[
-<p>%1$s puts you in control.</p>
-<p>Use it as a private browser:
-  <ul>
-    <li>Search and browse right in the app</li>
-    <li>Block trackers (or update settings to allow trackers)</li>
-    <li>Erase to delete cookies as well as search and browsing history</li>
-  </ul>
-</p>
-<p>%1$s is produced by Mozilla. Our mission is to foster a healthy, open Internet.<br/>
-<a href="%2$s">Learn more</a></p>
+<p>%1$s is a fast and lightweight browser produced by Mozilla. Our missions is to foster a healthy, open internet.<br/>
+<a href="%2$s">Learn more</a></p><br/><br/>
+<b><a href="%3$s" style="text-decoration:none">Support</a></b><br/><br/>
```

Comment:
can we put the bold and style into style.css file?

Reply:
ok